samples/detect.v1p4beta1.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * Copyright 2019, Google,LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable */
+
+'use strict';
+
+async function detectBatchAnnotateFiles(fileName) {
+  // [START vision_batch_annotate_files_beta]
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p4beta1;
```

Comment:
Can we just use this?

```js
const {ImageAnnotatorClient} = require('@google-cloud/vision');
```

Reply:
Do you mean we drop `v1p4beta1`? Or replace the variable `vision` with `{ImageAnnotatorClient}`?

Reply:
Kinda both :)   What's the default exported version from `index.js`?  If it has the properties you're looking to use, we could use the one liner.

Reply:
Removing v1p4beta1 leads to an error, but I changed the left side of = per your comment.

Reply:
> Can we just use this?

The `.v1p4beta1` is required, the explicit version is intentional for this beta feature.

We _drop_ the version only when using the default configured version of a package (which this is not).

The expected result of `require('@google-cloud/vision')` is to require the client for the GA `V1` version of the API (which it does). Keep the `v1p4beta1`, it's what we want.